.github/workflows/push.yml

```diff
@@ -18,8 +18,7 @@ jobs:
         - ./build-next.sh
         - ./build.sh develop
         docker_from:
-        - '' # use the default of the DOCKERFILE
-        - python:3.7-alpine
+        - '' # use the default of the build script
         - python:3.8-alpine
         # - python:3.9-rc-alpine # disable until Netbox's unit tests work
```

Comment:
Have you tried building 3.9? I believe the bug that was preventing it from working should have been fixed upstream.

Reply:
Not yet. I will enable it for testing.